ONSdigital/census-rm-sample-loader
# change printbatch start to be 0

**Motivation and Context**
To change the sample generator to start generating from print batch from 0

**What has changed**
samplegenerator will now start generating case from Printbatch 0

**How to test?**
Run sample generator and make sure the sample file generated have cases starting with printbatch '0'

Links
https://trello.com/c/6A8bwcof/1362-update-sample-file-generator-to-generate-printbatch-0-cases
